src/plugin/log/mod.rs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) The Swiboe development team. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See LICENSE.txt
+// in the project root for license information.
+use ::client;
+use ::error::Result;
+use ::plugin;
+use std::path;
+
+
+pub struct Plugin {
+    _client: client::Client,
```

Comment:
The _client is only for RAII. Rust compiler would give us a warning if the variable is not starts with an underscore.